language/functional_tests/tests/testsuite/natives/vector/vector_resource_copy.mvir

```diff
@@ -0,0 +1,15 @@
+module M {
+    import 0x0.Vector;
+
+    resource R {}
+
+    f() {
+        let v: Vector.T<Self.R>;
+        v = Vector.empty<Self.R>();
+        Vector.destroy_empty<Self.R>(copy(v));
+        Vector.destroy_empty<Self.R>(move(v));
+        return;
+    }
+}
+
+// check: COPYLOC_RESOURCE_ERROR
```

Comment:
Similar comment here--seems like this is testing that you can't copy a resource, which isn't related to the implementation of `destroy_empty`.